### DISCUSSION TITLE
请问对象存储里面每个文件夹最多会有多少文件？

### DISCUSSION THREAD

**Discussion by gdtv** [juicedata/juicefs#5037] [Q&A]
我使用的存储设备，它对每一层文件夹里（不算子文件夹）的文件数量有限制。
是否可以设置每一层文件夹里最大文件数量？

**Comment by zhijian-pro** [comment]
了解下 juicefs quota，通过 inode 去限制

**Reply by gdtv** [reply]
juicefs quota inode好像是指源文件不是指对象存储？
例如我将inode设置为1，意思是我只能上传一个文件，还是说只能在对象存储里存储一个“块”？ 我需要的是后者。
